.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
+language: go
+go:
+ - "1.13.x"
+script:
+  - go get ./...
+  - go get golang.org/x/lint/golint
+  - go get github.com/axw/gocov/gocov
+  - GO111MODULE=off go get gopkg.in/matm/v1/gocov-html
+  - golint  -set_exit_status ./...
+  - go vet ./...
```

Comment:
These commands are run as a part of `./scripts/test.sh` so I don't believe we need them here.